### DISCUSSION TITLE
Clarification for what this is for?

### DISCUSSION THREAD

**Discussion by wuast94** [autobrr/autobrr#1836] [Q&A] [answered]
As far as i understand this tool changes the normal arr workflow from sonarr radarr etc polling the indexers/feeds and check if something is needed to autobrr checking all the sources, sending everything to arr apps for further processing? 

the main thing as i understand is that we can check live in irc channels for new posts and not belonging to poll intervalls etc.

i am just using nzb not torrent so if i understand it right there is no real benifit in adding autobrr to the workflow right? 

sry for the maybee dumb questions but i try to understand this :)

EDIT: Great docs btw

**Comment by adapowers** [comment, marked as answer]
@wuast94 I think that's correct. If you're 100% Usenet, the only thing autobrr might give you is the ability to set a custom poll interval,  but a) you wouldn't want to set it too low, or risk getting in trouble with your indexers and b) there's no real advantage in doing so anyway, since being "first" to get a file doesn't matter the way it does in torrent-land.

**Reply by adapowers** [reply]
It's still great that autobrr HAS Usenet support, since it allows people who use a mix of torrents/NZBs to fully transition download checking to autobrr and disable it on the Sonarr/Radarr side. But at its heart, autobrr is built for people who want to jump on torrent files ASAP for ratio-building purposes.

**Reply by adapowers** [reply]
I should add, to say, the one place where it COULD be handy for your workflow is if you want to use filters to download things other than your monitored shows/movies. But I don't think that's a hugely common case, especially for someone just on Usenet.

Also note, I am not a dev, so take what I'm saying here with a grain of salt. :) I could be way off-base.

**Reply by wuast94** [reply]
thanks for the detailed explaination :) the filter part could be intresting yes :)

but anyways could this be a good one to get my foot into torrents, but this will be when i have time, safty first ;)
